packages/badge/components/ColorCodedBadge.tsx

```diff
@@ -42,4 +43,4 @@ const ColorCodedBadge: React.FC<ColorCodedBadgeProps> = ({
   );
 };
 
-export default ColorCodedBadge;
+export default React.memo(ColorCodedBadge);
```

Comment:
sorry I know this is annoying. This one wasn't memoized before so doesn't need it here